unstructured/ingest/cli/cmds/mongodb.py

```diff
@@ -25,32 +27,36 @@ def get_cli_options() -> t.List[click.Option]:
                 "mongos instance to connect to, or a list of hostnames",
             ),
             click.Option(["--port"], type=int, default=27017),
-        ]
-        return options
-
-
-@dataclass
-class MongoDBCliWriteConfig(MongoDBWriteConfig, CliConfig):
-    @staticmethod
-    def get_cli_options() -> t.List[click.Option]:
-        options = [
             click.Option(
                 ["--database"], type=str, required=True, help="database name to connect to"
             ),
             click.Option(
                 ["--collection"], required=True, type=str, help="collection name to connect to"
             ),
+            click.Option(
+                ["--batch-size"],
+                default=100,
+                type=click.IntRange(0),
+                help="how many records to read at a time per process",
+            ),
```

Comment:
this option only applies to the source connector, right? ...in which case this probably belongs somewhere other than MongoDBCliConfig otherwise both the source and destination connectors have this option?

Reply:
Yep. moving it to a ReadConfig